### ISSUE TITLE
Getting "XInputReader-32.dll: %1 is not a valid Win32 application" after exporting runnable jar from eclipse

### ISSUE THREAD

**Issue by seannybgoode** [StrikerX3/JXInput#2]
As the title says. Exporting from eclipse with the "Package required libraries into the jar" option, as I normally do. When it goes to load XInputDevice it throws that exception. The libraries are both inside the jar, and in the /<myapp>/lib/native directory. Running Java version 1.8.0_40.

Edit: I should add that the libraries run fine inside eclipse, but dies when I try to run the exported jar. I *really* want that rumble pack support, so I'm dying to make this work.

I've tried to resolve this, but I'm not having any luck.

**Comment by seannybgoode** [comment]
Fixed this. The problem was with the copying approach to XInputDevice used to load native library files. I packaged the project into a jar file, and specified a path to the dlls included in the maven project in eclipse, which I copied to my project. Then I linked the jar the way I would any other library. 

Here's the code changes I made. They're in the static code block in that class.

 final String arch = System.getProperty("os.arch").contains("64") ? "64" : "32";
        try {
            String path = new String(System.getProperty("user.dir") + "\\lib\\native\\" + "XInputReader-" + arch + ".dll");
            System.load(path);
        } catch (final UnsatisfiedLinkError e) {
            throw new Error("XInputDevice could not find required library: XInputReader-" + arch, e);
        }

**Comment by StrikerX3** [comment]
I just made big changes to how the native libraries are loaded. Hopefully they'll solve this problem.
I tested this by creating a new Maven project, adding JXInput as a dependency, writing some simple code just to check if the devices are connected and then exporting with the "Package required libraries into jar" option as you said above. The resulting jar executed succesfully with `java -jar test.jar`.
